br.com.wiselabs.simplexity.appdata.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Franklin Ronald -->
+<component type="desktop-application">
+    <id>br.com.wiselabs.simplexity</id>
+    <metadata_license>0BSD</metadata_license>
+    <project_license>BSD-3-Clause</project_license>
+    <name>Simplexity</name>
+    <summary>Simple desktop app for Perplexity.AI</summary>
+    <developer id="br.com.wiselabs">
+        <name>Wiselabs Software</name>
+    </developer>
+    <description>
+        <p>The non-nonsense simple desktop app for Perplexity AI. It's just work, no magic, no tray, no scripts injected and nothing obfuscated.</p>
```

Comment:
can you add some more description?

Reply:
Description updated. File moved to upstream, check it out: https://github.com/Wiselabs/simplexity/blob/master/metadata/br.com.wiselabs.simplexity.appdata.xml